### PR TITLE
drive_mirror: Redistribute boot_target_image to meet cases description.

### PR DESCRIPTION
**Pull request by vivianQizhu**
Redistribute boot_target_image to meet cases description.
id: 1146789, 1330404
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
@yanan-fu Would you help review this? Thanks.

**Comment by yanan-fu** [comment]
The following code can get default velue, but i think it is better to define  defult "boot_target_image=no" in cfg too.
`tests/drive_mirror_complete.py:44:        if params.get("boot_target_image", "no") == "yes":`

**Comment by vivianQizhu** [comment]
@yanan-fu Thanks, but I prefer not define it, since only drive_mirror_complete use this parameter, most cases in the cfg file do not use it.

**Comment by yanan-fu** [comment]
@vivianQizhu  i think is it more safe to define in the cfg too.  only suggestion ^^
and i am not very clearly about the future, any new case will use this param? 

@xutian @CongLi do you have any advise about this issue ?

**Comment by vivianQizhu** [comment]
@yanan-fu If any other case will use it, it definitely will use it with drive_mirror_complete.py, so to define it inside the code is safe enough, no need to do duplicate default key definition. I think the logic is good enough, and our code style from the documentation is "Keep it simple."

**Comment by yanan-fu** [comment]
Acked-by: Yanan Fu yfu@redhat.com

**Comment by xutian** [comment]
@vivianQizhu  better to comment you configuration,  target  image will not be boot because default valus for 'boot_target_image' is 'no'.

**Comment by vivianQizhu** [comment]
@xutian Thanks, updated.

File: qemu/tests/cfg/drive_mirror.cfg [hunk 1]
```diff
@@ -34,14 +34,14 @@
     tmp_dir = "/tmp"
     repeat_times = 3
     cancel_timeout = 5
+    #For drive_mirror_complete, target image will not be boot by default, as boot_target_image=no
```
Comment (xutian) [inline qemu/tests/cfg/drive_mirror.cfg]: still need 
monitors = "main  status",
monitor_type = qmp
main_monitor = main_monitor

Comment (vivianQizhu) [inline qemu/tests/cfg/drive_mirror.cfg]: @xutian thanks, but not only drive_mirror need this, I prefer add them together for all block jobs later, handle this separately.

Comment (xutian) [inline qemu/tests/cfg/drive_mirror.cfg]: Ok.

**Pull request merged**